### ISSUE TITLE
Option to ignore properties

### ISSUE THREAD

**Issue by NGumby** [madskristensen/TypeScriptDefinitionGenerator#12]
It would be nice to be able to ignore some properties from being generated.  For example

```
public class Person
{
    public string FullName { get { return $"{FirstName} {LastName}"; } }
    public string LastName {get; set;}
    public string FirstName {get; set;}
}
```

I would like to ignore FullName from being generated, maybe with an Attribute.

**Comment by styfle** [comment]
I think it would be cool if all of the [JSON.NET Attributes](http://www.newtonsoft.com/json/help/html/SerializationAttributes.htm#JsonIgnoreAttribute) were observed. Then you could do something like the following:

```cs
public class Person
{
    [JsonIgnore]
    public string FullName { get { return $"{FirstName} {LastName}"; } }

    [JsonProperty("lastName")]
    public string LastName { get; set; }

    [JsonProperty("firstName")]
    public string FirstName { get; set; }
}
```

This would emit:

```ts
declare module server {
    interface Person {
        lastName: string;
        firstName: string;
    }
}

```

**Comment by raboud** [comment]
the attribute [IgnoreDataMember] works

**Comment by raboud** [comment]
JsonIgnore should now be recognized.
Please Try Build 1.1.24 to see if your issues are corrected.

**Comment by nhammadi** [comment]
Issue resolved with the latest version (1.1.24) @madskristensen you should close this issue
